modules/core/include/opencv2/core/bindings_utils.hpp

```diff
@@ -20,6 +20,18 @@ CV_EXPORTS_W String dumpInputOutputArray(InputOutputArray argument);
 
 CV_EXPORTS_W String dumpInputOutputArrayOfArrays(InputOutputArrayOfArrays argument);
 
+CV_EXPORTS_W String dumpBool(bool argument);
```

Comment:
Lets don't expose this as an exported API of core module.

Please use "static inline" for these simple test wrappers.